core/src/components/slider/slider.ts

```diff
@@ -250,6 +266,41 @@ const computeCleanValue = (value: number, min: number, max: number, intStepSize:
 	);
 };
 
+/**
+ * Method to update the dirtyValues for the slider keyboard navigation
+ * @param handleIndex - index of the handle to update
+ * @param _dirtyValues$ - writable signal that contains dirtyValues
+ * @param values - slider values
+ * @param stepSize - slider step size
+ * @param updateDirection - if equals 1 - value is increased, if equals -1 values is decreased
+ */
+const updateDirtyValue = (
+	handleIndex: number,
+	_dirtyValues$: WritableSignal<number[]>,
+	values: number[],
+	stepSize: number,
+	updateDirection: number,
+) => {
+	_dirtyValues$.update((value) => {
+		value = [...value];
```

Comment:
why don't we just define a new const instead of reusing the same name? I think it would be more readable

Reply:
you mean _dirtyValue$? as it is still a writable signal it makes sense to keep the old notation as it is in the same core file and user knows to what it pertains

Reply:
I meant const newValue = [...value]; and so on